docs/configuration.rst

```diff
@@ -1663,6 +1663,26 @@ Example:
    The created ``needs.json`` file gets stored in the ``outdir`` of the current builder.
    So if ``html`` is used as builder, the final location is e.g. ``_build/html/needs.json``.
 
+.. _needs_build_needumls:
+
+needs_build_needumls
+~~~~~~~~~~~~~~~~~~~~
+
+Creates a folder during other builds, like ``html``.
+
+This option like :ref:`needs_build_json` allows to have one single Sphinx-Build for two output formats. 
```

Comment:
This option works like :ref:`needs_build_json`.

Rest of the sentence can be deleted, as it is written already above.
dont' mention that there are two output format,s that sound confusing.
Just write, what it does. It exports needuml data.